devtools/docker-setup.md

```diff
@@ -0,0 +1,231 @@
+# Dockerizing Notes
+
+## Table of Contents
+
+- [Quick-Start](#quick-start)
+  - [Requirements](#quick-start-requirements)
+  - [Running](#quick-start-running)
+- [Full Setup](#full-setup)
+  - [Requirements](#full-setup-requirements)
+  - [Running](#full-setup-running)
+  - [Running One-Offs](#full-setup-running-one-offs)
+    - [Frontend One-Offs](#frontend-one-offs)
+    - [Backend One-Offs](#backend-one-offs)
+    - [UAA One-Offs](#uaa-one-offs)
+    - [CF One-Offs](#cf-one-offs)
+- [Tear Down](#tear-down)
+- [Side Effects](#side-effects)
+
+## Quick Start
+
+For quick and low impact changes (e.g. typos, changes to **existing** styling)
+or for a quick glance at the dashboard, you can follow this quick start
+section to start the frontend testing server.
+For all others changes, use the [full setup guide](#full-setup).
+
+### Quick-Start: Requirements
+
+- [docker-compose](https://docs.docker.com/compose/install/)
+
+### Quick-Start: Running
+
+```sh
+docker-compose up frontend_testing_server
+```
+
+It will say the server is up when everything is done. At the point, you can
+navigate to `http://localhost:8001` and see the testing server.
```

Comment:
i added the note here: https://github.com/18F/cg-dashboard/pull/1036/commits/1abf66766cde43cef5e684687bd3ee24fec7a749
 
in the quick-start, you don't have to login (since it doesn't use the real CF API).

Let me know your thoughts